iceoryx_examples/iceoptions/README.md

```diff
@@ -81,6 +84,10 @@ If we want to enforce the contract that the publisher needs to support a `histor
 `requirePublisherHistorySupport` to `true`. In this case the subscriber will only connect if the publisher history support is at least as large as is request.
 By default this is set to `false` and best-effort behavior is used.
 
+!!! warning
+    In case of n:m communication, the history feature will **not** provide the overall last n samples based on delivery point in time!
+    For more information about this limitation see the [QoS article](https://iceoryx.io/v2.0.0/concepts/qos-policies/).
```

Comment:
Please set this to `latest` and let a script do the work for you ... be lazy :laughing:

Reply:
I will save a script some execution time and leave it here. We need to be frugal with resources! :grin:

Reply:
This makes the script more complex and will break for v3. Please keep it uniform on master

Reply:
IMHO the script will need this complexity anyway. Imagine we are on `master` at version `v2.1.16` and want to create both a `v2.1.17` on a release branch and `v3.0.0`. 

cc @dkroenke

Reply:
My idea would be, everything points to master and then we start the magic script with a release tag and the links are updated.

Reply:
@elfenpiff exactly

Reply:
Ok, done.

Reply:
@mossmaurice would it be possible to have a snapshot of master on the homepage? Not as default but selectable in the combo box, e.g.
```
latest <- this is the default; maybe instead of `lastest` the version number like it is now
master
v2.0.0
v1.0.1
v1.0.0 
```
What do you think? This is nothing for v2 but maybe afterwards

Reply:
@elBoberido Sure, we just need to setup a CI job for that!

Reply:
Created the issue: https://github.com/eclipse-iceoryx/iceoryx/issues/1201